### PR TITLE
Fix a few nits in tests and automation. Don't try to wordcount ls-rem…

### PR DESCRIPTION
**Pull request by Pike**
…ote.

Nits in tests are around undoing changes made to the environment,
and to not modify the checkout.

These are my take on #545 . I also included one more thing I found not mentioned there, which is extending the unittests workflow on all dev branches.

### CODE REVIEW

File: __tests__/git.test.ts [hunk 1]
```diff
@@ -192,40 +204,44 @@ describe('git', () => {
...
-        token: '123',
-        pusher: {
-          name: 'asd',
-          email: 'as@cat'
-        },
-        clean: true,
-        cleanExclude: '["cat", "montezuma"]',
-        workspace: 'other',
-        isTest: TestFlag.NONE
+    describe('', () => {
```
Comment (JamesIves) [inline __tests__/git.test.ts:216]: Can we add a description to this suite?

**Review by JamesIves — APPROVED** [review]
Looks good, just the one comment!

File: __tests__/git.test.ts (hunk 1 above)
Comment (Pike) [reply __tests__/git.test.ts]: Better?

Comment (JamesIves) [reply __tests__/git.test.ts]: Perf

**Pull request merged**